pallets/phala/src/mining.rs

```diff
@@ -447,7 +447,7 @@ pub mod pallet {
 			let who = ensure_signed(origin)?;
 			let pubkey = Self::ensure_miner_bound(&miner)?;
 			let worker =
-				registry::Workers::<T>::get(&pubkey).ok_or(Error::<T>::WorkerNotRegistered)?;
+				registry::Workers::<T>::get(pubkey).ok_or(Error::<T>::WorkerNotRegistered)?;
```

Comment:
Let's skip the `mining.rs` and `stakepool.rs`, said h4x.
See: https://github.com/Phala-Network/phala-blockchain/pull/1005#discussion_r1002984896

Reply:
I can revert clippy commit, but it will made CI fail, maybe you loosen clippy rule on master then I sync master and revert clippy commits?

Reply:
I tend to let it fail until the big PR merged. The clippy is not required to pass in the merge condition Settings.

Reply:
Reverted cargo clippy for phala pallet, I think clippy for pruntime and phactory is safe, revert them will have to handle conflict, so I keep those